chalice/awsclient.py

```diff
@@ -179,6 +186,19 @@ def _is_settling_error(self, error):
             return True
         return False
 
+    def invoke_function(self, name, context=None, payload=None):
+        # type: (str, _OPT_STR, bytes) -> Dict[str, Any]
+        kwargs = {
+            'FunctionName': name,
+            'InvocationType': 'RequestResponse',
+        }
+        if payload is not None:
+            kwargs['Payload'] = payload
+
+        return self._call_client_method_with_retries(
+            self._client('lambda').invoke, kwargs, max_attempts=1
```

Comment:
I might have missed this in the review comments but what's the point of using `_call_client_method_with_retries` with a retry of 1?

Reply:
Is there an alternative? The method I was using didn't have that but got removed when I rebased. It looked like the expectation was to use that method. I don't really see a reason to re-write the old method when it was deleted in favor of this one.

Reply:
>Is there an alternative?

I still like I'm missing something, but I'm specifically asking what's the reason for not just using `self._client('lambda').invoke(**kwargs)`?

The call client method with retries is for retries, and if you have a `max_attempts` of 1 that's effectively a noop is it not?

> It looked like the expectation was to use that method. 

I touched on this in my previous comment:

> That method was added specifically because there's some settling time when making config changes (roles, event sources). We shouldn't need this here.


If there's not a good reason to use it, I'd prefer to just keep the code simpler.